assets/app/styles/_messages.less

```diff
@@ -160,27 +164,20 @@ ul.messenger-theme-flat .messenger-message .messenger-actions a {
 ul.messenger-theme-flat {
   .messenger-message.alert-error {
     &:before {
-      content: "\e609";
+      content: "\e61c";
```

Comment:
@rhamilto it would be really awesome down the road if patternfly could generate a variables file with variable names for the icons that match the class name (or some sane conversion) and then set to the data content for that icon.  For cases like this where we need to inject our icon into markup we don't control using css.  Then we wouldn't have to worry about updating this every time we update patternfly.